styles/globals.css

```diff
@@ -2,7 +2,7 @@
 @import url(./fonts/fonts.css);
 body {
   padding: 0;
-  margin: 0 5vw;
+  /* margin: 0 5vw; */
```

Comment:
If we are no longer need this style, please remove the line. Please no commented put code.